apps/fabric-website/src/components/Site/customizations.ts

```diff
@@ -4,7 +4,7 @@ import { IAppCustomizations, IExampleCardCustomizations } from '@uifabric/exampl
 
 const exampleCardCustomizations: IExampleCardCustomizations[] = [
   { title: 'Fluent', customizations: FluentCustomizations },
-  { title: 'MDL2', customizations: DefaultCustomizations }
+  { title: 'Pre-Fluent', customizations: DefaultCustomizations }
```

Comment:
Hm... fair that MDL2 is probably unclear to most audiences, but "Pre-Fluent" feels a bit odd. What about "Default"? I recognize it might be unintuitive that the website demos are applying the Fluent theme by default, but it feels it'd be more meaningful. Thoughts?

Reply:
"Default" seems reasonable. I'll update it.